sdk/README.md

```diff
@@ -50,61 +60,70 @@ We are running the tests over approximately 80+ Pipelines spread across differen
+      ? Value for param `url1` of type `string`? (Default  is `gs://ml-pipeline-playground/shakespeare1.txt`) gs://ml-pipeline-playground/shakespeare1.txt
+      ? Value for param `url2` of type `string`? (Default is `gs://ml-pipeline-playground/shakespeare2.txt`) gs://ml-pipeline-playground/shakespeare2.txt
  
       Pipelinerun started: parallel-pipeline-run-th4x6
-
     ```
    
    We will see the logs of the running Tekton Pipeline streamed, similar to the one below
       
       ```bash
       Waiting for logs to be available...
 
-      [gcs-download-2 : gcs-download-2] I find thou art no less than fame hath bruited And more than may be gatherd by thy shape Let my    
-      presumption not provoke thy wrath
-      [gcs-download : gcs-download] With which he yoketh your rebellious necks Razeth your cities and subverts your towns And in a moment         makes them desolate
-      [echo : echo] Text 1: With which he yoketh your rebellious necks Razeth your cities and subverts your towns And in a moment makes           them desolate
-      [echo : echo] Text 2: I find thou art no less than fame hath bruited And more than may be gatherd by thy shape Let my presumption not 
-      provoke thy wrath
+      [gcs-download-2 : gcs-download-2] I find thou art no less than fame hath bruited And more than may be gatherd by thy shape Let my presumption not provoke thy wrath
+      [gcs-download : gcs-download] With which he yoketh your rebellious necks Razeth your cities and subverts your towns And in a moment makes them desolate
+      [echo : echo] Text 1: With which he yoketh your rebellious necks Razeth your cities and subverts your towns And in a moment makes them desolate
+      [echo : echo] Text 2: I find thou art no less than fame hath bruited And more than may be gatherd by thy shape Let my presumption not provoke thy wrath
       ```
+
       
 ## Build Tekton from Master
 
-In order to utilize the latest features and functions the team has been driving in Tekton, we suggest that Tekton must be built from [master](https://github.com/tektoncd/pipeline/blob/master/DEVELOPMENT.md#install-pipeline). Features that require special builds different from the 'Tested Version' will be listed below.
-      
-## Test Kubeflow Pipelines with Tekton
+In order to utilize the latest features and functions of the `kfp-tekton` compiler, we suggest to install Tekton from a
+nightly built or build it from the [master](https://github.com/tektoncd/pipeline/blob/master/DEVELOPMENT.md#install-pipeline) branch. 
+Features that require special builds different from the 'Tested Version' will be listed below.
 
-Please [refer to the instructions here](./python/tests/README.md) as you work on a PR test sample Kubeflow Pipelines in their test data folder to ensure your PR is improving the number of successful samples
 
-## Experimental features
+## Experimental Features
```

Comment:
small nit: can we rename it to something like `Additional Features` for now? We will rework this section when we migrate to the kfp api.

Reply:
sure